.tekton/check-no-quay-io-references.yaml

```
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: check-no-quay-io-references
+  namespace: rh-acs-tenant
+spec:
+  description: This task is used to ensure that no quay.io references are present in the catalog files,
+    as they are not allowed in the production environment.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the artifact with
+      the application source code. This should be the result of the git-clone task,
+      results from other tasks might fail as dirty.
```

I wonder how does this keeps surviving.
```suggestion
      the application source code.
```